.infrastructure/helm-chart/todoapp/values.yaml

```diff
@@ -0,0 +1,51 @@
+common:
+ namespace: todoapp
+
+todoapp:
+  image:
+    name: todoapp
+    repository: ikulyk404/todoapp
+    version: 4.0.1
+
+  rollingUpdate:
+    maxSurge: 1
+    maxUnavailable: 1
+
+  configs:
+    PYTHONUNBUFFERED: "1"
+
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: "150m"
+    limits:
+      memory: "256Mi"
+      cpu: "150m"
+
+  affinity:
+    weight: 1
+    matchExpressions:
+      key: app
+      operator: In
+      values: kube2py
+
+  hpa:
+    minReplicas: 2
+    maxReplicas: 5
+    cpu:
+      averageUtilization: 70
+    memory:
+      averageUtilization: 70
+
+  secret:
+    SECRET_KEY: QGUyKHl4KXYmdGdoM19zPTB5amEtaSFkcGVieHN6XmRnNDd4KS1rJmtxXzN6Zio5ZSoK(@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*)
```

Comment:
Do not store secret value as a plain text in values.yml

Reply:
I have a question to this part. 
In video "Creating Helm Chart - Working with values files" ([27 sec](https://mate.academy/learn/kubernetes-basics/helm-practice?section=video&videoId=7093)), mentor implemented it in exect way. 
Could you say how it must be implemented? I need to use values in b64?

Reply:
Or create a 3rd file with secretKeyRef on secret.yml. and that values copy to values.yaml, after paste to deployment.yml?

Reply:
@YuriiSmolii here is example of second variant (https://ververica.zendesk.com/hc/en-us/articles/360015021359-How-to-secure-passwords-secrets-in-values-yaml-with-Kubernetes-secrets)